kernel/safe_typing.ml

```diff
@@ -1301,7 +1301,9 @@ let import lib cst vodigest senv =
   mp,
   { senv with
     env;
-    modresolver = Mod_subst.add_delta_resolver mb.mod_delta senv.modresolver;
+    (* Do NOT store the name quotient from the dependencies in the set of
+       constraints that will be marshalled on disk. *)
```

Comment:
Could you then add a comment telling that you hijack paramresolver precisely to prevent the imported resolver to be marshalled again?

Reply:
Congrats for the -3.8% on CoLoR in passing!

Reply:
I have tweaked the comments describing these fields at the top of the file, is that fine with you?

Reply:
OK